Formula/c/crystal.rb

```diff
@@ -147,7 +146,8 @@ def install
     end
 
     # Install crystal
-    crystal_install_dir.install ".build/crystal"
+    libexec.install ".build/crystal"
+    (crystal_install_dir/"crystal").write_env_script libexec/"crystal", LD_RUN_PATH: "${LD_RUN_PATH:-}:#{HOMEBREW_PREFIX}/lib"
```

Comment:
```suggestion
    if OS.mac?
      crystal_install_dir.install ".build/cystal"
    else
      libexec.install ".build/crystal"
      (crystal_install_dir/"crystal").write_env_script libexec/"crystal", LD_RUN_PATH: "${LD_RUN_PATH:-}:#{HOMEBREW_PREFIX}/lib"
    end
```
This is only needed on Linux, no?